CoreFoundation/Base.subproj/CFFileUtilities.c

```diff
@@ -1023,7 +1024,7 @@ CF_PRIVATE void _CFIterateDirectory(CFStringRef directoryPath, Boolean appendSla
     if (!CFStringGetFileSystemRepresentation(directoryPath, directoryPathBuf, CFMaxPathSize)) return;
     
 #if DEPLOYMENT_TARGET_WINDOWS
-#error this path does not support calculateFullResultPath but it must do so someday
+#warning this path does not support calculateFullResultPath but it must do so someday
```

Comment:
We need a SR for this.

Reply:
I would love to do that.  I am afraid that I do not fully understand what the warning is about.  I would have just implemented it otherwise.

Reply:
@millenomi - could you please explain what `calculateFullResultPath` actually means?

Reply:
Spoke offline, this is vestigial, and can be safely removed.